### PR TITLE
zh-translation: 1.3.x/announcing-1.3/change-notes/index.md #1582

### PR DESCRIPTION
**Pull request by gorda**
- [x] Docs

Ref: [ServiceMesher#1582](https://github.com/servicemesher/istio-official-translation/issues/1582)

### CODE REVIEW

File: content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
+- **添加**。为使用双向 TLS 的服务添加了信任域验证。默认情况下，服务器仅对来自同一信任域的请求进行身份验证。
+- **添加**。添加了一些[标签](/zh/docs/ops/configuration/mesh/secret-creation/)，其用于按命名空间控制服务帐户密码的生成。
+- **添加**。添加了 SDS 支持，以实现向每个 Istio 控制平面服务传递私钥和证书。
+- **添加**。为 Citadel 添加了对[自检](/zh/docs/ops/diagnostic-tools/controlz/)支持。
+- **添加**。为 15014 端口的 Citadel Agent 的 `/metrics`  endpoint 添加了指标，用于监控 SDS 服务。
+- **添加**。使用 8080 端口上的 `/debug/sds/workload` 和 `/debug/sds/gateway` 向 Citadel Agent 添加了诊断程序。
+- **改进**。改进了 ingress gateway，以实现使用 SDS 时[从另一个 secret 加载受信任的 CA 证书](/zh/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-TLS-ingress-gateway)。
+- **改进**。通过强制使用 [Kubernetes Trustworthy JWT](/zh/blog/2019/trustworthy-jwt-sds) 改进了 SDS 的安全性。
+- **改进**。通过统一日志记录模式，改进了 Citadel Agent 日志记录。
+- **移除**。移除对 [1.13 之前版本 Kubernetes](/zh/blog/2019/trustworthy-jwt-sds) 的 Istio SDS 支持。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]:
```suggestion
- **移除**。移除对 [Kubernetes 1.13 之前版本](/zh/blog/2019/trustworthy-jwt-sds) 的 Istio SDS 支持。
```

File: content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 2]
```diff
@@ -1,83 +1,83 @@
...
+- **移除**。暂时移除与 Vault CA 的集成。SDS 的一些要求导致了本次临时移除，但我们将在之后的版本中重新引入 Vault CA 集成。
+- **启用**。默认情况下启用 Envoy JWT 过滤器以提高安全性和可靠性。
+
+## 遥测{#telemetry}
+
+- **添加**。为 Envoy gRPC 添加了访问日志服务 [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) 的支持。
+- **添加**。为 Citadel 监控添加了一个 Grafana 仪表盘。
+- **添加**。为 sidecar 注入 webhook 监控添加了[指标](/zh/docs/reference/commands/sidecar-injector/#metrics)。
+- **添加**。添加了控制平面指标，用于监控 Istio 的配置状态。
+- **添加**。添加了针对目标是 `Passthrough` 和 `BlackHole` 集群的流量的遥测报告。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]:
```suggestion
- **添加**。添加了流向目标是 `Passthrough` 和 `BlackHole` 集群的流量的遥测报告。
```

File: content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 3]
```diff
@@ -1,83 +1,83 @@
...
+- **移除**。移除了不推荐使用的 `Adapter` 和 `Template` 自定义资源（CRD）。
+- **弃用**。弃用可用于产生 API 属性的 HTTP API 规范。我们将在 Istio 1.4 中移除对生成 API 属性的支持。
+
+## 策略{#policy}
+
+- **改进**。改进了速率限制的实现，当配额后端不可用时，仍允许通信。
+
+## 配置管理{#configuration-management}
+
+- **修复**。修复了  阻止过多 gRPC ping 关闭连接的情况。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]:
```suggestion
- **修复**。修复了阻止过多 gRPC ping 关闭连接的情况。
```

File: content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 4]
```diff
@@ -1,83 +1,83 @@
...
 
-## Miscellaneous
+## 杂项{#miscellaneous}
 
-- **Added** new images based on [distroless](/zh/docs/ops/configuration/security/harden-docker-images/) base images.
-- **Improved** the Istio CNI Helm chart to have consistent versions with Istio.
-- **Improved** Kubernetes Jobs behavior. Kubernetes Jobs now exit correctly when the job manually calls the `/quitquitquit` endpoint.
+- **添加**。添加了基于 [distroless](/zh/docs/ops/configuration/security/harden-docker-images/) 基础镜像的新镜像。
+- **改进**。改进了 Istio CNI Helm 图表，使其具有与 Istio 一致的版本。
+- **改进**。改进了 Kubernetes Job 的行为。当 Job 手动调用 `/quitquitquit` endpoint 时，Kubernetes Job 现在可以正确退出。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]:
```suggestion
- **改进**。改进了 Kubernetes Job 的行为。当 Job 手动调用 `/quitquitquit` endpoint 时，Kubernetes Job 现在可以正常退出。
```

File: content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 5]
```diff
@@ -1,83 +1,83 @@
...
-- **Improved** Galley to avoid control plane upgrade failures.
+## 安装{#installation}
+
+- **添加** 实验性的[清单和配置文件命令](/zh/docs/setup/install/istioctl/)，用于安装和管理 Istio 控制平面，以便进行评估。
+
+## 流量管理{#traffic-management}
+
+- **添加**。当未根据 Istio 的[约定](/zh/docs/ops/deployment/requirements/)命名端口时，添加了对 HTTP、TCP 出站流量[自动检测协议](/zh/docs/ops/configuration/traffic-management/protocol-selection/)的功能。
+- **添加**。为实现双向 TLS 操作，添加了一个模型至 Gateway API。
+- **修复**。修复了当服务首先以宽容双向 TLS 模式通过网络与 MySQL 和 MongoDB 之类的协议进行通信时出现的问题。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]: 不通顺，建议修改

Comment (gorda) [reply content/zh/news/releases/1.3.x/announcing-1.3/change-notes/index.md]: @yuxiaobo96 感谢细心 review，相关问题已修改

**Pull request merged**